Gem/Assets/ViewModels/AssetsFilterViewModel.swift

```diff
@@ -55,7 +55,7 @@ struct AssetsFilterViewModel {
 
     var networksModel: NetworkSelectorViewModel {
         NetworkSelectorViewModel(
-            items: chainsFilter.allChains,
+            state: .loaded(chainsFilter.allChains),
```

Comment:
```suggestion
            state: .data(chainsFilter.allChains),
```

can we change from `.loaded` to `.data` makes more sense when I look at tit